01_fib/marksomething/main_test.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestFibStringPositive(t *testing.T) {
+
+	testCases := []struct {
```

Comment:
Consider using map rather than slice for test cases (key being the test case name) - see [Dave Cheney post](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests).

Reply:
That is much more elegant, have now used the t.Run() method as well so that the individual cases show up in the `go test -v` output